includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -48,42 +48,62 @@ public function admin_menus() {
 	}
 
 	/**
+	 * The theme "extra" should only be shown if the current user can modify themes
+	 * and the store doesn't already have a WooCommerce compatible theme.
+	 */
+	protected function should_show_theme_extra() {
+		return (
+			current_user_can( 'install_themes' ) &&
+			current_user_can( 'switch_themes' ) &&
+			! is_multisite() &&
+			! current_theme_supports( 'woocommerce' )
```

Comment:
Why? Maybe the user has a theme that happens to somewhat support WooCommerce, but if he's building a store, shouldn't we recommend Storefront anyway?

Reply:
This logic should be unchanged from the dedicated Storefront step.